src/main/scala/com/softwaremill/realworld/common/CustomHttpAuthorization.scala

```diff
@@ -0,0 +1,51 @@
+package com.softwaremill.realworld.common
+
+import sttp.model.HeaderNames
+import sttp.model.headers.WWWAuthenticateChallenge
+import sttp.tapir.{Codec, CodecFormat, DecodeResult, EndpointInput, Mapping, Schema, header}
+
+// This authorization allows the use of both the "Bearer" header and the "Token" parameter at the beginning of the token.
+object CustomHttpAuthorization {
```

Comment:
I think we are dealing with *authentication* here

Reply:
How about `CustomAuthInputs`?